docs/rackhd/install_os.rst

```diff
@@ -249,8 +249,11 @@ installPartitions   Array            *optional*   *installPartitions* is to spec
 kvm                 Boolean          *optional*   The value is *true* or *false* to indicates to install kvm or not, default is **false**. (**ESXi, PhotonOS** doesn't support this parameter)
 switchDevices       Array            *optional*   **(ESXi only)** If specified, this contains an array of objects with switchName and uplinks (optional) parameters. If *uplinks* is omitted, null or empty, the vswitch will be created with no uplinks. See switchDevices_ for more details.
 postInstallCommands Array            *optional*   **(ESXi only)** If specified, this contains an array of string commands that will be run at the end of the post installation step.  This can be used by the customer to tweak final system configuration.
-installType         String           *optional*   **(PhotonOS only)** The value is *minimal* or *full* to indicate the type of installed OS, defualt *installType* is **minimal**
-installScriptUri    String           *optional*   The download URI for a custom kickstart/preseed/autoyast/cloud-config template to be used for automatic installation/configuration.
+installType         String           *optional*   **(PhotonOS only)** The value is *minimal* or *full* to indicate the type of installed OS, defualt *installType* is **minimal**
+gdbPort             String           *optional*   **(ESXi only)** Serial port used for GDB debug
+logPort             String           *optional*   **(ESXi only)** Serial port for log output in ESXi
```

Comment:
Please add a valid example here. like "com0", "com1", or a specified link as how to set gdbPort and logPort.